### PR TITLE
get_string_pattern_with_prefix short str pattern modified

### PR DESCRIPTION
**Pull request by hbusul**
These are the changes to fix the bug described at [issue 288](https://github.com/python-rope/rope/issues/288) . It is passing all the tests and also solving the issue. Could you take a look at it please?

### CODE REVIEW

File: ropetest/codeanalyzetest.py [hunk 1]
```diff
@@ -342,6 +342,15 @@ def test_is_on_function_keyword(self):
         finder = worder.Worder(code)
         self.assertTrue(finder.is_on_function_call_keyword(len(code) - 1))
 
+    def test_get_word_range_with_fstring(self):
+        code = 'auth = 8\nmy_var = f"some value {auth}"\nprint(auth)\nother_val = "some other"'
+        finder = worder.Worder(code)
+        self.assertEqual(finder.get_word_range(45), (45, 49))
+
+    def test_get_word_range_with_fstring2(self):
```
Comment (soupytwist) [inline ropetest/codeanalyzetest.py]: nit: This test is a bit unnecessary.

**Comment by soupytwist** [comment]
Looks great and makes sense. It's not a huge deal, but if you can, please remove the redundant test here to keep down on clutter.

Will merge soon regardless. Thanks for the fix!

**Comment by hbusul** [comment]
For sure, removed the second test.

**Comment by soupytwist** [comment]
Thanks again!

**Pull request merged**